src/js/study-browser/eva-study-browser-widget-panel.js

```diff
@@ -420,6 +427,7 @@ EvaStudyBrowserWidgetPanel.prototype = {
         params = _.omit(params, ['genome']);
         _this._updateColumns(params)
         EvaManager.get({
+            host:'http://ves-ebi-f8:8080/eva-812/webservices/rest/',
```

Comment:
I guess we have to remember to set this back to the main webservices when we deploy a new release

Reply:
It the `browsable` flag in use by the website at the moment? Otherwise this can just point to the standard wwwint URL.

Reply:
removed